nipype/utils/config.py

```diff
@@ -114,13 +116,29 @@ def set_log_dir(self, log_dir):
         """
         self._config.set('logging', 'log_directory', log_dir)
 
-    def get(self, section, option):
-        return self._config.get(section, option)
+    def get(self, section, option, default=None):
+        if option in CONFIG_DEPRECATIONS:
+            msg = ('Config option "%s" has been deprecated as of nipype %s. Please use '
+                   '"%s" instead.') % (option, CONFIG_DEPRECATIONS[option][1],
+                                       CONFIG_DEPRECATIONS[option][0])
+            warn(msg)
+            option = CONFIG_DEPRECATIONS[option][0]
```

Comment:
This is a message directed at users, but it's the nipype dev who's going to request the old versions of these. I think we need a warning for users that their config file contains a deprecated option.

Perhaps in `__init__`, you can do at the end:

```Python
for option in CONFIG_DEPRECATIONS:
    if self._config.has_option(option):
        # Warn
        new_option = CONFIG_DEPRECATIONS[option][0]
        if not self._config.has_option(new_option):
            self._config.set(new_option, self._config.get(option))
```

Or similar.

Reply:
Done